src/NuGet.Clients/NuGet.VisualStudio.Common/NuGetUIThreadHelper.cs

```diff
@@ -16,7 +16,7 @@ public static class NuGetUIThreadHelper
         /// Initially it will be null and will be initialized to CPS JTF when there is CPS
         /// based project is being created.
         /// </summary>
-        private static Lazy<JoinableTaskFactory> _joinableTaskFactory;
+        private static Lazy<JoinableTaskFactory> LazyJoinableTaskFactory;
```

Comment:
This one is still private. Do we really need to change name?

Reply:
IDE complains about it because it does not match naming conventions used in rest of the code.